libsoftwarecontainer/softwarecontainer_network.conf.in

```diff
@@ -1,2 +1,3 @@
 lxc.network.type = veth
 lxc.network.link = lxcbr0
```

Comment:
Maybe we should set the name "lxcbr0" via a variable that defaults to "lxcbr0"? This way it is configurable by the integrators.